cache/util/getRetryStrategy.js

```diff
@@ -0,0 +1,12 @@
+// simple retry strategy
+
+// default max delay in milliseconds
+const DEFAULT_MAX_DELAY = 10000
```

Comment:
Not really giving up after 10 secs, but at the longest period, retrying after 10 secs. I suppose that's still short though, I can bump the max up.